Realm/RealmSwift/Object.swift

```diff
@@ -62,6 +62,10 @@ public class Object : RLMObjectBase, Equatable {
         super.init()
     }
 
+    public class func createInDefaultRealmWithObject(object: AnyObject) -> Self {
```

Comment:
I think I had purposely left this out because it doesn't significantly simplify the general case in Swift:

```swift
MyObject.createInDefaultRealmWithObject(obj)
MyObject.createInRealm(defaultRealm(), withObject: obj)
```